src/collections/_documentation/development/sdk-dev/event-payloads/exception.md

```diff
@@ -85,6 +85,7 @@ mechanism-specific values.
 
 : Optional flag indicating whether the user has handled the exception
   (for example, via `try ... catch`).
+  When this flag is set to `false`, it probably means that the Application has been terminated.
```

Comment:
```suggestion
   Where applicable, when this flag is set to `false` it indicates the application terminated.
```
Perhaps we could go futher and give examples like, JS event doesn't mean the app crashed since it's a webpage on the browser. On a native C++ application, that would terminate the app.

Reply:
@bruno-garcia I'm all for it, let's discuss it again on TSC about which languages should be the examples and happy to change.